### PR TITLE
Made pixi 5 conversion of scissor mask implementation.

### PR DESCRIPTION
**Pull request by J00nz**
##### Pre-Merge Checklist

- [x] Tests and/or benchmarks are included
- [x] Documentation is changed or added
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

**Comment by GoodBoyDigital** [comment]
You rock! thanks for this PR! Do you have an example of this working with scissor rect we I can check out? Cheers!

File: packages/core/test/Renderer.js [hunk 1]
```diff
@@ -82,4 +82,57 @@ describe('PIXI.Renderer', function ()
...
+            this.renderer.destroy();
+            this.renderer = null;
+        });
+
+        it('should have scissor-masks enabled', function ()
+        {
+            expect(this.renderer.mask.enableScissor).to.equal(true);
+        });
+
+        it('should use scissor masks whith axis aligned squares', function ()
```
Comment (rin3s) [inline packages/core/test/Renderer.js]:
```suggestion
        it('should use scissor masks with axis aligned squares', function ()
```

**Comment by bigtimebuddy** [comment]
ping @J00nz, could you please provide a live example of this feature working?

**Comment by ivanpopelyshev** [comment]
I will deal with that as soon as I'm available :)

**Comment by GoodBoyDigital** [comment]
scratch that, noticed that `enableScissor` was false :D

File: packages/core/src/mask/MaskSystem.js [hunk 2]
```diff
@@ -80,7 +80,7 @@ export default class MaskSystem extends System
         }
         else if (this.enableScissor
             && !this.scissor
-            && this.renderer._activeRenderTarget.root
+            // && this.renderer._activeRenderTarget.root
```
Comment (GoodBoyDigital) [inline packages/core/src/mask/MaskSystem.js]: we should replace this line with: 
```&& !this.renderer.renderTexture.current```
this makes sure we only use the stencil mask if we are on the root.

Comment (J00nz) [reply packages/core/src/mask/MaskSystem.js]: 👍

**Pull request merged**